x-pack/plugins/security_solution/public/common/components/event_details/summary_view.tsx

```diff
@@ -61,6 +61,7 @@ const fields = [
   { id: 'user.name' },
   { id: SOURCE_IP_FIELD_NAME, fieldType: IP_FIELD_TYPE },
   { id: DESTINATION_IP_FIELD_NAME, fieldType: IP_FIELD_TYPE },
+  { id: 'Ransomware.files.path', label: 'Impacted Files' },
```

Comment:
Is the purpose of this line to show the ransomware impacted files in the `Summary` tab of the Alert Details flyout? I'm currently not seeing it there.
![image](https://user-images.githubusercontent.com/616158/108184054-c1b45580-70d8-11eb-9067-22e51651a7dc.png)

Reply:
This was changed with the search strategy refactor i think? So I just took it out for now, we can put it back in the follow-up